httpclient/src/test/java/feign/httpclient/ApacheHttpClientTest.java

```diff
@@ -68,7 +68,7 @@ public void queryParamsAreRespectedWhenBodyIsEmpty() throws InterruptedException
     @Path("/withBody")
     public String withBody(@QueryParam("foo") String foo, String bar);
 
-    @PUT
+    @GET
```

Comment:
I would prefer that we keep the `@PUT` test case and add the `@GET` test case.  I am surprised that the `@PUT` test case was testing without a body, that's not really valid anyway.

Could you please restore the `@PUT` and correct it to send a body?

Reply:
Sure, I'll do that tomorrow. Thank you for feedback.

Reply:
Yeah, agree with Kevin on this one.

Add a new test and leave existing as is.

Reply:
After some tries, I cannot figure out how to keep PUT with `@QueryParam` and `UrlEncodedFormEntity` by depending on `RequestBuilder`. For now, I kind of give up. Sorry for no step forward.